examples/example_server_forum.py

```diff
@@ -0,0 +1,155 @@
+import socket
+import sys
+import socketserver
+from http.server import BaseHTTPRequestHandler
+from http.server import HTTPServer
+import urllib.parse
+
+# using port numbers prepended with 9 to avoid calling sudo during test
+servport = 9090
+posts_file = "./forum/posts.txt"
+
+def jsonify(user, msg):
+    max_chars = 256
+    num_attributes = 2
+    max_chars -= (num_attributes*6+1) # "":"", and {} but last line has no comma
+    return "{\"user\":\"" + user + "\",\"msg\":\"" + msg + "\"}";
+
+# I did this in case any messages have newline characters (so I won't necessarily store comments as one line the posts file)
+def read_simple_json(json_str, start_idx):
```

Comment:
I'd recommend checking out the json module to see if that makes this easier. Maybe something like: `json.loads(json_str)`

Reply:
Right now I am still using the read_simple_json, but I can perhaps try switching to json.loads now that I am taking the approach of reading the whole file into a json array (in server.getCommentsFromUser()). What I have right now works for the inputs I've tried, so I'm not too worried about it right now.
Example inputs (for the value of "msg" in the json):
1. {} gonna break? "} \"}
2. What did the cat say to the ghost? "You're creeping meowt!"